server/poll/poll.go

```diff
@@ -135,7 +143,7 @@ func (p *Poll) getVotedAnswers(userID string) ([]string, error) {
 	for _, o := range p.AnswerOptions {
 		for _, v := range o.Voter {
 			if userID == v {
-				votedAnswer = append(votedAnswer, o.Answer)
+				votedAnswer = append(votedAnswer, p.getAnswerOptionName(o))
```

Comment:
Why did you fix this bug on the server side? The `(X)` suffix feels like a formnating thing that the API shouldn't return.

How hard is it to fix the bug on the webapp only?

Reply:
Before this changes, `(X)` suffix as the part of answer text was added on the server side [here](https://github.com/matterpoll/matterpoll/pull/304/files#diff-67c60e0c08d1bd5cd27b356d9f8ef017L66), and the logic to change selected button color is to check if answer text between PostAction and Metadata is same. So it would be better to add `(X)` suffix in Metadata on the server side, which is close to where suffix in PostAction is added.

And I think it's little hard to fix this on webapp, because webapp doesn't have information on voters other than myself, and can't count the number of voters. With regexp, it might be work fine, but I feel little weird.

Reply:
I you think it's harder to fix on the webapp I'm fine with the current fix